vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CSharpApp.targets

```diff
@@ -28,7 +28,7 @@
     <Copy SourceFiles="%(AppXManifest.Identity)" 
           DestinationFiles="$(MSBuildProjectDirectory)\$(OutputPath)Appx\%(AppxManifest.PackagePath)" 
           Condition="'%(AppxManifest.SubType)'!='Designer'"/>
-    <Exec Command="powershell -NonInteractive -NoProfile -Command Add-AppxPackage -Register $(MSBuildProjectDirectory)\$(OutputPath)Appx\AppxManifest.xml" 
+    <Exec Command="powershell -NonInteractive -NoProfile -Command Add-AppxPackage -Register $(MSBuildProjectDirectory)\$(OutputPath)Appx\AppxManifest.xml -DependencyPath $(AppxDependenciesPath)\*" 
```

Comment:
What happens when -DependencyPath is passed without a location? i.e. when building from VS and AppxDependenciesPath is not set?

Reply:
@dannyvv  This target doesn't run when building from VS, it's only used by `react-native run-windows` to deploy the app from layout